inference/nlp/realtime/llm/bloom_176b/djl_deepspeed_deploy.ipynb

```diff
@@ -0,0 +1,613 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.                "TENSOR_PARALLEL_DEGREE": "8",
</pre><p>Do you want to talk about this <code>TENSOR_PARALLEL_DEGREE</code>  parameter in the markdown cell above? What's it going to do? Should the customer play with this value?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3642/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>sure added details of this is and how to leverage and also some helpful links</p>